studio/hooks/misc/withAuth.tsx

```diff
@@ -50,7 +50,7 @@ export function withAuth<T>(
       },
     })
 
-    const isLoggedIn = Boolean(session)
+    const isLoggedIn = IS_PLATFORM ? Boolean(session) : true
```

Comment:
do we still need this check actually since we're technically doing this check within lib/auth by returning a default session if IS_PLATFORM